docs/introduction-to-smart-contracts.rst

```diff
@@ -472,19 +478,20 @@ receives the address of the new contract on the stack.
 
 .. index:: selfdestruct
 
-Self-destruct
-=============
+Deactivate and Self-destruct
+============================
 
-The only possibility that code is removed from the blockchain is
-when a contract at that address performs the ``selfdestruct`` operation.
-The remaining Ether stored at that address is sent to a designated
-target and then the storage and code is removed from the state.
+The only way to remove code from the blockchain is when a contract at that address performs the ``selfdestruct`` operation. The remaining Ether stored at that address is sent to a designated target and then the storage and code is removed from the state. Removing the contract and 'cleaning it up' sounds like a good idea, but the operation doesn't really clean up. If you send Ether to removed contracts, the Ether is forever lost.
```

Comment:
"the operation doesn't really clean up" - why not? Perhaps rather frame this as "sounds like a good idea, but might be dangerous: If you ..."

Reply:
@chriseth Yeah, I didn't really like the wording copied over either :)